src/Admin/FieldDescription.php

```diff
@@ -105,16 +105,6 @@ public function getValue($object)
             $object = $this->getFieldValue($object, $parentAssociationMapping['fieldName']);
         }
 
-        $fieldMapping = $this->getFieldMapping();
-        // Support embedded object for mapping
-        // Ref: https://www.doctrine-project.org/projects/doctrine-orm/en/latest/tutorials/embeddables.html
-        if (isset($fieldMapping['declaredField'])) {
-            $parentFields = explode('.', $fieldMapping['declaredField']);
-            foreach ($parentFields as $parentField) {
-                $object = $this->getFieldValue($object, $parentField);
-            }
-        }
```

Comment:
This is not needed anymore (and will be buggy instead) if the correct fieldName is set in `ModelManager::getNewFieldDescriptionInstance`.